README.md

```diff
@@ -31,6 +31,21 @@ The repository has the following permanent branches:
  * **develop** This contains the latest code. All the contributing PRs must be sent to this branch. When we want to release the next version of the app, this branch is merged into the `master` branch.
 
  * **apk** This branch contains the apks for the code in the develop branch. The apks are automatically updated when a commit is pushed to `develop` branch.
+ 
+## Features
```

Comment:
Maybe this session would be better before setting up the project, also without a header.
Since this is part of the introduction to the app